src/frontend/packages/core/src/shared/components/add-service-instance/add-service-instance/add-service-instance.component.spec.ts

```diff
@@ -5,45 +5,40 @@ import { TabNavService } from '../../../../../tab-nav.service';
 import { BaseTestModulesNoShared } from '../../../../../test-framework/cloud-foundry-endpoint-service.helper';
 import { ServicesService } from '../../../../features/service-catalog/services.service';
 import { ServicesServiceMock } from '../../../../features/service-catalog/services.service.mock';
-import { AppNameUniqueDirective } from '../../../app-name-unique.directive/app-name-unique.directive';
```

Comment:
The sorting of these imports seems different to the one's I'm seeing. Do you use vs code, and if so do you use the TypeScript Hero plugin or have any custom rules? We've probably got something wrong in our docs here either way so would be good to get to the bottom of